include/dlaf/schedulers.h

```diff
@@ -0,0 +1,42 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2020-2021, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <pika/execution.hpp>
+#include <pika/modules/resource_partitioner.hpp>
+#include <pika/runtime.hpp>
+#include <pika/thread.hpp>
+
+#ifdef DLAF_WITH_CUDA
+#include <pika/modules/async_cuda.hpp>
+#endif
+
+#include <dlaf/init.h>
+#include <dlaf/types.h>
+
+namespace dlaf {
```

Comment:
TODO: This might not need to be public... it's currently only used inside `transform` which is internal. Opinions?

Reply:
I've moved it now to `dlaf::internal` but can move it back if someone prefers that.

Reply:
Feel free to reopen this if someone feels differently.